Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM alpine:3.1
+FROM gliderlabs/alpine:3.3
 ENTRYPOINT ["/bin/resolvable"]
 
+RUN apk add --update -t build-deps go git mercurial
```

Comment:
Maybe `--no-cache` instead of `--update` as it will have the same effect of updating first but will also not store the index files, making the image slightly slimmer.